x11-themes/golden-xcursors/golden-xcursors-0.8.ebuild

```diff
@@ -14,31 +18,40 @@ IUSE=""
 RDEPEND=""
 DEPEND="${RDEPEND}"
 
+S="${WORKDIR}/${MY_P:5}"
+
+DOC_CONTENTS="To use this set of cursors, edit or create the file ~/.Xdefaults
+and add the following line:
+Xcursor.theme: Gold
+
+You can change the size by adding a line like:
+Xcursor.size: 48
+
+Also, to globally use this set of mouse cursors edit the file:
+	  /usr/share/cursors/xorg-x11/default/index.theme
+and change the line:
+	Inherits=[current setting]
+to
+	Inherits=Gold
+
+Note this will be overruled by a user's ~/.Xdefaults file."
+
+DOCS=(README)
```

Comment:
`README` is automaticall added to the `DOCS` array. Please remove the array definition.

Reply:
Done.